src/main/java/org/ow2/proactive/workflow_catalog/rest/entity/WorkflowRevision.java

```diff
@@ -241,4 +241,8 @@ public String getLayout() {
         return layout;
     }
 
+    @Override
+    public int compareTo(Object o) {
+        return createdAt.compareTo(((WorkflowRevision) o).createdAt);
```

Comment:
Just a question: Wouldn't it be better to implement Comparator which compares createdAt like a WorkflowRevisionCreatedAtComparator class?